orbit/pkg/table/fleetd_logs/fleetd_logs.go

```diff
@@ -0,0 +1,166 @@
+package fleetd_logs
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/osquery/osquery-go/plugin/table"
+	"github.com/rs/zerolog"
+)
+
+// No timezone, always return in UTC. Use this format because SQLite3
+// knows how to parse it.
+// See https://www.sqlite.org/lang_datefunc.html
+const timeFormatString = "2006-01-02 15:04:05.999999999"
+
+var Logger = logger{}
+var MaxEntries uint = 10_000
+
+func TablePlugin() *table.Plugin {
+	columns := []table.ColumnDefinition{
+		table.TextColumn("time"),
+		table.TextColumn("level"),
+		table.TextColumn("payload"),
+		table.TextColumn("message"),
+		table.TextColumn("error"),
+	}
+
+	return table.NewPlugin("fleetd_logs", columns, generate)
+}
+
+func generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	output := []map[string]string{}
+
+	for _, entry := range Logger.logs {
+		row := make(map[string]string, 5)
+		// It would be nice if we could return NULL instead of an
+		// empty string when the error is empty
+		row["time"] = entry.Time
+		row["level"] = entry.Level.String()
+		row["payload"] = string(entry.Payload)
```

Comment:
This makes sense to me, I can store it as string. It will also stop it from re-interpreting the byte array of each record every time this table is queried